bugbot/rules/not_landed.py

```diff
@@ -167,15 +182,12 @@ def handle_attachment(self, attachment, res):
 
     def get_patch_data(self, bugs):
         """Get patch information in bugs"""
-        nightly_pat = Bugzilla.get_landing_patterns(channels=["nightly"])[0][0]
 
         def comment_handler(bug, bugid, data):
             # if a comment contains a backout: don't nag
             for comment in bug["comments"]:
                 comment = comment["text"].lower()
-                if nightly_pat.match(comment) and (
-                    "backed out" in comment or "backout" in comment
-                ):
+                if "backed out" in comment or "backout" in comment:
```

Comment:
This is going to have many false positives. Why do we need to remove `nightly_pat.match(comment) `?

Reply:
I think I got confused with the way it was checking the patches as some were marked to be not backed out when they appeared to be. After some additional investigation, I realize that removing it unnecessary: 85801acd844ef0dbae2a0c3233e63b93d75eb6f1